test/playwright.fixtures.ts

```diff
@@ -114,6 +115,10 @@ fixtures.browserType.init(async ({ playwright, browserName }, run) => {
 }, { scope: 'worker' });
 
 fixtures.browser.init(async ({ browserType, browserOptions }, run) => {
+  if (process.env.PWMODE === 'android') {
```

Comment:
you should be able to use `mode` here, also you can probably throw?

Reply:
The Playwright fixtures should be quite in-depended since they get used in playwright-test, so I think its better to do it like that with env vars.